src/fermilib/utils/_jellium.py

```diff
@@ -150,36 +150,31 @@ def momentum_vector(momentum_indices, grid_length, length_scale):
     return momentum_vector
 
 
-def momentum_kinetic_operator(n_dimensions, grid_length,
-                              length_scale, spinless=False):
+def momentum_kinetic_operator(grid, spinless=False):
     """Return the kinetic energy operator in momentum second quantization.
 
     Args:
-        n_dimensions: An int giving the number of dimensions for the model.
-        grid_length: Int, the number of points in one dimension of the grid.
-        length_scale: Float, the real space length of a box dimension.
-        spinless: Bool, whether to use the spinless model or not.
+        grid (fermilib.utils.Grid):
+        spinless (bool): Whether to use the spinless model or not.
 
     Returns:
-        operator: An instance of the FermionOperator class.
+        FermionOperator:
```

Comment:
It was an information-preserving transform, but yeah it does look pretty empty now. Done.